specification/azuredata/resource-manager/Microsoft.AzureData/preview/2019-07-24-preview/azuredata.json

```diff
@@ -1863,6 +2386,261 @@
         }
       }
     },
+    "PageOfDataControllerResource": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "items": {
+            "$ref": "#/definitions/DataControllerResource"
+          },
+          "type": "array"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to retrieve next page of results."
+        }
+      }
+    },
+    "DataControllerResource": {
+      "description": "Data controller resource",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/DataControllerProperties",
+          "description": "null",
```

Comment:
Can you add proper description or remove it?

Reply:
Added a simple generic description.